packages/cypress/src/junit.ts

```diff
@@ -130,7 +131,7 @@ function appendReplayUrlsToFailureNodes(node: INode, replayUrls: string[]) {
       failure.children[0] = escapeForXml(output);
     });
   } catch (e) {
```

Comment:
These `e` and `err` names are pet peeves of mine 😆 especially when they cause us to spend more characters renaming them